### PR TITLE
Refactor/ros control code style

### PR DESCRIPTION
**Pull request by SammyRamone**
refactores ros control to our code style

### CODE REVIEW

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h [hunk 1]
```diff
@@ -58,35 +58,35 @@ struct Joint
...
   bool read();
   void write();
   void setParent(hardware_interface::RobotHW* parent);
 
 private:
-  ros::NodeHandle _nh;
+  ros::NodeHandle nh_;
 
-  void syncWritePWM();
+  void syncWritePwm();
```
Comment (jgueldenstein) [inline bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: Since PWM is an abbreviation, i would keep it as syncWritePWM

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h [hunk 2]
```diff
@@ -58,35 +58,35 @@ struct Joint
...
 
   bool loadDynamixels(ros::NodeHandle& nh);
-  bool writeROMRAM(ros::NodeHandle& nh);
+  bool writeRomram(ros::NodeHandle& nh);
```
Comment (jgueldenstein) [inline bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: same as PWM

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h [hunk 3]
```diff
@@ -58,35 +58,35 @@ struct Joint
...
   bool stringToControlMode(std::string control_mode_str, ControlMode &control_mode);
   void switchDynamixelControlMode();
   diagnostic_msgs::DiagnosticStatus createServoDiagMsg(int id, char level, std::string message, std::map<std::string, std::string> map);
-  void processVTE(bool success);
+  void processVte(bool success);
```
Comment (jgueldenstein) [inline bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: same as PWM and ROMRAM

File: bitbots_ros_control/src/dynamixel_controller.cpp [hunk 4]
```diff
@@ -7,24 +7,24 @@ namespace dynamixel_controller
...
     // get handles for joints
-    for(unsigned int i=0; i<n_joints_; i++)
+    for(unsigned int i=0; i<n_joints; i++)
     {
         try
         {
-        joints_.push_back(hw->getHandle(joint_names_[i]));
-        _joint_map[joint_names_[i]] = i;          
+        joints.push_back(hw->getHandle(joint_names[i]));
+          joint_map_[joint_names[i]] = i;
```
Comment (jgueldenstein) [inline bitbots_ros_control/src/dynamixel_controller.cpp]: this indentation seems off to me

File: bitbots_ros_control/src/node.cpp [hunk 5]
```diff
@@ -30,7 +30,6 @@ int main (int argc, char *argv[]){
   ros::Rate rate(pnh.param("control_loop_hz", 200));
 
   while (ros::ok()){
-    bool read_sucessfull = hw.read();
```
Comment (jgueldenstein) [inline bitbots_ros_control/src/node.cpp]: why is this deleted, we still need to read the hardware, we just don't care anymore if it was successful

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h (hunk 1 above)
Comment (SammyRamone) [reply bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: okay

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h (hunk 2 above)
Comment (SammyRamone) [reply bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: okay

File: bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h (hunk 3 above)
Comment (SammyRamone) [reply bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h]: okay
